client/src/graphics/graphics_util.lua

```diff
@@ -143,11 +143,11 @@ local maxQuadPool = 100
 function GraphicsUtil:newRecycledQuad(x, y, width, height, sw, sh)
   local result = nil
   if #self.quadPool == 0 then
-    result = love.graphics.newQuad(x, y, width, height, sw, sh)
+    result = love.graphics.newQuad(x, y, width, height, sw or width, sh or height)
   else
     result = self.quadPool[#self.quadPool]
     self.quadPool[#self.quadPool] = nil
-    result:setViewport(x, y, width, height, sw, sh)
+    result:setViewport(x, y, width, height, sw or width, sh or height)
```

Comment:
Don't worry about these, I was mainly getting annoyed at the extremely long lines for initializing recycled quads accessing `self.theme.images.IMG_multibar_stop_bar:getWidth()` several times previously.
So in the new `Stack:initializeGraphics()` it's readable without scrolling to the right if the editor can make use of a full monitor width.